doozer/doozerlib/cli/release_gen_payload.py

```diff
@@ -767,37 +773,63 @@ async def sync_payloads(self):
             filtered_payload_entries = {}
             for payload_tag_name, payload_entry in payload_entries.items():
                 if payload_entry.rhcos_build is None and payload_entry.build_inspector.is_under_embargo():
+                    if self.runtime.assembly_type is AssemblyTypes.STANDARD and not self.cve_exclusion_ack:
+                        raise IOError(f"Embargoed build detected for payload tag {payload_tag_name} with "
+                                      f"payload_entry: {payload_entry} \n --cve-exclusion-ack must be specified, "
+                                      f"if this was intended")
+                    # Otherwise
                     # We do not want embargoed builds to get leaked to public quay
                     # RHCOS builds are considered non-embargoed for now
                     continue
```

Comment:
I think this can be simplified if at this point, we just don't consider the build private, and let it be included in the payload entries from the start, while noting that there are private entries, e.g.


```
                    # (note RHCOS builds can't be embargoes for now)
                    if self.runtime.assembly_type is not AssemblyTypes.STANDARD:
                        # Unless releasing, we do not want embargoed builds synced to public quay
                        continue
                    if self.cve_exclusion_ack:
                        # consciously releasing private content
                        self.includes_embargo = True
                        # probably create an AssemblyIssue somehow here to be permitted by the assembly?
                    else:
                        raise IOError(f"Embargoed build detected for payload tag {payload_tag_name} with "
                                                f"payload_entry: {payload_entry} \n --cve-exclusion-ack must be specified, "
                                                f"if this was intended")
```

Reply:
> probably create an AssemblyIssue somehow here to be permitted by the assembly?
I was thinking we can have a flag in build sync?